# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1300563189 25200
#      Sat Mar 19 12:33:09 2011 -0700
# Node ID 30ee8f0cc7c2e1e08eb41f1e73b45aa179c0b624
# Parent  f8ea3eabcb85fc015c56ea52c43217465cf723b9
Add dump_prefs to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-19  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/basics.txi: Add dump_prefs to documentation.
+
 2010-03-19  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/octave.texi, interpreter/plot.txi: Add daspect, pbaspect
 	functions to documentation.
 
 2010-03-19  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/octave.texi, interpreter/plot.txi: Add getappdata,
diff --git a/doc/interpreter/basics.txi b/doc/interpreter/basics.txi
--- a/doc/interpreter/basics.txi
+++ b/doc/interpreter/basics.txi
@@ -331,16 +331,21 @@ Octave searches for @file{.octaverc}.
 If you start Octave in your home directory, commands from the file
 @file{~/.octaverc} will only be executed once.
 @end table
 
 A message will be displayed as each of the startup files is read if you
 invoke Octave with the @option{--verbose} option but without the
 @option{--silent} option.
 
+The @code{dump_prefs} function is useful for determining what customizations
+to Octave are possible and which are in effect.
+
+@DOCSTRING(dump_prefs)
+
 @node Quitting Octave
 @section Quitting Octave
 @cindex exiting octave
 @cindex quitting octave
 
 @DOCSTRING(quit)
 
 @DOCSTRING(atexit)
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-19  Rik  <octave@nomad.inbox5.com>
+
+	* miscellaneous/dump_prefs.m: Improve docstring.
+
 2010-03-19  Rik  <octave@nomad.inbox5.com>
 
 	* plot/daspect.m, plot/pbaspect.m: Use newline between @deftypefnx
 	macros to get correct appearance.
 
 2011-03-19  Ben Abbott  <bpabbott@mac.com>
 
 	* geometry/trisurf.m: Set default edgecolor as Matlab does. Add demos.
diff --git a/scripts/miscellaneous/dump_prefs.m b/scripts/miscellaneous/dump_prefs.m
--- a/scripts/miscellaneous/dump_prefs.m
+++ b/scripts/miscellaneous/dump_prefs.m
@@ -12,20 +12,21 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {} dump_prefs (@var{file})
-## Have Octave dump all the current user preference variables to
-## @var{file} in a format that can be parsed by Octave later.  If
-## @var{file} is omitted, the listing is printed to stdout.
+## @deftypefn {Function File} {} dump_prefs ()
+## @deftypefn {Function File} {} dump_prefs (@var{fid})
+## Dump all of the current user preference variables in a format that can be
+## parsed by Octave later.  @var{fid} is a file descriptor as returned by
+## @code{fopen}.  If @var{file} is omitted, the listing is printed to stdout.
 ## @end deftypefn
 
 ## Author: jwe
 
 function dump_prefs (file)
 
   if (nargin == 0)
     file = stdout;
